functions/supplier/debezium-supplier/src/main/java/org/springframework/cloud/fn/supplier/debezium/DebeziumProperties.java

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright 2023-2023 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.fn.supplier.debezium;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import io.debezium.engine.format.SerializationFormat;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+/**
+ * @author Christian Tzolov
+ */
+@ConfigurationProperties("debezium")
+public class DebeziumProperties {
+
+	public enum DebeziumFormat {
+		/**
+		 * JSON change event format.
+		 */
+		JSON(io.debezium.engine.format.JsonByteArray.class, "application/json"),
+		/**
+		 * AVRO change event format.
+		 */
+		AVRO(io.debezium.engine.format.Avro.class, "application/avro"),
+		/**
+		 * ProtoBuf change event format.
+		 */
+		PROTOBUF(io.debezium.engine.format.Protobuf.class, "application/x-protobuf"),;
+
+		private final Class<? extends SerializationFormat<byte[]>> serializationFormat;
+		private final String contentType;
+
+		DebeziumFormat(Class<? extends SerializationFormat<byte[]>> serializationFormat, String contentType) {
+			this.serializationFormat = serializationFormat;
+			this.contentType = contentType;
+		}
+
+		public Class<? extends SerializationFormat<byte[]>> serializationFormat() {
+			return serializationFormat;
+		}
+
+		public final String contentType() {
+			return contentType;
+		}
+	};
+
+	/**
+	 * Spring pass-trough wrapper for debezium configuration properties. All properties with a 'debezium.inner.*' prefix
+	 * are native Debezium properties.
+	 */
+	private Map<String, String> inner = defaultConfig();
```

Comment:
In other cases we just use a name `properties`.
See `KafkaProperties.properties` for example.

Reply:
It is important to streass that those are native Debezium configuration vs. Spring properitieds. 
'Inner' better conveys this meaning.